stable/dex-controller/values.yaml

```diff
@@ -9,7 +9,7 @@ controller:
     resources:
       limits:
         cpu: 100m
-        memory: 100Mi
+        memory: 512Mi
```

Comment:
this is a bump of 5x's is this based on the current configurration of a default konvoy cluster or is this for a particular large node count topology?

Reply:
@aescobar the issue has happened in the past. I saw this problem with smaller cluster sizes. The size plays a role here but it is not the main motivation.

The main issue comes when dex-controller crashes for the first time with OOM, after that moment is hard to start it again because during the bootstrap operation crashes again. The bootstrap action requires a lot of `memory`. We saw this behavior in the past with different cluster sizes, but this time we tuned the memory limit to accommodate to that bootstrap action.

Second motivation is that `memory is cheap` and allocating 512Mi is nothing for us, in return of a better user experience.

Third, I saw this issue happening `more` often when using kommander and attaching clusters. However I have to say I also faced during our early MWTs.

Reply:
Alright, thank you for answering! If this affects our current cluster then it makes sense. I was just worried that it was just for large Node cases and it wasnt worth the change, but it sounds like it is.

Reply:
Yes, let's say if the memory limit for big clusters would have been 9G then obviously I won't adapt it, but the required value to have a steady behavior seems quite low for us.